lib/tasks/encrypt.rake

```diff
@@ -24,33 +24,37 @@ end
 end
 
 namespace :db do
+  def modify_encryptable_fields(klass, action)
+    klass.order(:id).each do |auth_source_ldap|
+      auth_source_ldap.encryptable_fields.each do |field|
+        str = auth_source_ldap.read_attribute(field.to_sym)
+        auth_source_ldap.update_column(field.to_sym,
+                                       auth_source_ldap.send("#{action}_field", str))
+      end
+    end
+  end
+
+  namespace :auth_sources_ldap do
+    desc "Encrypt LDAP authentication source fields"
+    task :encrypt => :environment do
+      modify_encryptable_fields(AuthSourceLdap, :encrypt)
+    end
+
+    desc "Decrypt compute resource fields"
+    task :decrypt => :environment do
+      modify_encryptable_fields(AuthSourceLdap, :decrypt)
+    end
+  end
+
   namespace :compute_resources do
```

Comment:
This CR-specific rake task was a mistake IMHO, could you make it one generic "encrypt everything" task and have the old CR task redirect to that?  It's idempotent, so safe to bundle everything together.

We can then update packaging to call that one task instead of listing a task for each thing we encrypt.  (e.g. https://github.com/theforeman/foreman-packaging/blob/foreman-1.8.0-1/foreman/foreman.spec#L721)